client/dashboard/domains/domain-connection-setup/components/style.scss

```diff
@@ -0,0 +1,39 @@
+@import "@wordpress/base-styles/variables";
+
+.progress-step {
+	&__circle {
+		width: 24px;
+		height: 24px;
+		border-radius: $radius-round;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		font-size: 12px;
+		font-weight: 500;
+		border: 1px solid;
+		background-color: transparent;
+		border-color: $gray-200;
+		color: $gray-600;
+
+		&--completed {
+			background-color: var( --dashboard__background-color-success );
+			border-color: var( --dashboard__background-color-success );
+			color: #fff;
+			fill: #fff;
```

Comment:
nit: it would be better to use `$white` since we use color variables in this file